### DISCUSSION TITLE
Kubernetes support

### DISCUSSION THREAD

**Discussion by Trebuac** [nextflow-io/nextflow#2128] [Q&A]
Hi,
I'm new to nextflow and was interesting by the kubernetes integration. I see in nextflow documentation that k8s support is "experimental" ([in executor section](https://www.nextflow.io/docs/latest/executor.html#kubernetes) and [kuberun cli command](https://www.nextflow.io/docs/latest/cli.html#kuberun) but not on the dedicated [Kubernetes page](https://www.nextflow.io/docs/latest/kubernetes.html)).
What does this involve ?
Is the feature is stable enough and you planned to release it officially in the next version of nextflow ? Are there any major changes planned ?
Can it be used in a production ready environment ?

Thanks

Clément

**Comment by pditommaso** [comment]
Thanks for reporting this, the docs was a bit inconsistent at this regard. The Kubernetes executor and support is considered stable, however in respect to the `kuberun` command, we realised it has some limitation due to the complex nature of K8 deployment and it cannot support all Nextflow features. it should only be considered as a quick start for evaluation. 

The docs should be more clear at this regard now: https://www.nextflow.io/docs/latest/kubernetes.html#limitation
